compiler/tests/pil.rs

```diff
@@ -1,7 +1,8 @@
-use number::FieldElement;
 use std::path::Path;
 
-pub fn verify_pil(file_name: &str, query_callback: Option<fn(&str) -> Option<FieldElement>>) {
+use number::GoldilocksField;
+
+pub fn verify_pil(file_name: &str, query_callback: Option<fn(&str) -> Option<GoldilocksField>>) {
```

Comment:
Does this need to be GL here?

Reply:
I think so as this calls pilcom which is goldilocks specific?

Reply:
but does the type need to be concrete here?

Reply:
Right, it does not need to. But now if you try to use it with Bn you get a compilation error. Whereas if we make it generic, you'd get nonsense out of pilcom, which is more confusing?

Reply:
yea  let's leave it